documentation/new-version-quickstart.md

```diff
@@ -97,10 +97,10 @@ Authentication
 
 Once the environment is setup, all you need to do is to create an authenticated client. Before creating a client, you will first need to authenticate to Azure. In specific, you will need to provide a credential for authenticating with the Azure service.  The `azidentity` module provides facilities for various ways of authenticating with Azure including client/secret, certificate, managed identity, and more.
 
-Our default option is to use **DefaultAzureCredential** which will make use of the environment variables we have set and take care of the authentication flow for us.
+We will use **EnvironmentCredential** to authenticate to Azure which will make use of the environment variables we have set and take care of the authentication flow for us. For further usage of `azidentity`, you could refer to [this document](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity#readme)
 
 ```go
-cred, err := azidentity.NewDefaultAzureCredential(nil)
+cred, err := azidentity.NewEnvironmentCredential(nil)
```

Comment:
As i know, most of the quick start, samples are using NewDefaultAzureCredential, so what is the major reson to switch to  NewEnvironmentCredential?

Reply:
I'm intent to use environment credential to explain the environment variable set in the previous part. I'll check other quick start to see if introduce the azlogin process is much better.